sourcegraph/about
# Update new case study layout

Closes #5540. Refreshes new Case Study layout (only applied to Nutanix right now). This is in preparation for standardizing the other Case Study pages.

### Changelog
1. Updates style changes to `NewCaseStudyLayout`
2. Combines `BlockquoteWithBorder` & `InContentBlockquote` into `Blockquote` since they are so similar. (I will update the Handbook with this change with #5513).
3. Moves bg image assets used in `GatedResourceLayout` to the public directory since they're beginning to be shared amongst components. This simplifies imports on `GatedResourceLayout` too.

### Note
- Once all case study layouts are standardized, `NewCaseStudyLayout` and `CaseStudyLayout` will merge.

### Test
1. Ensure prettier has standardized the proposed changes.
2. Nav to `/case-studies/nutanix-fixed-log4j-with-sourcegraph` and ensure refresh design matches Figma
3. Ensure no breaking changes to any page hero backgroudn using `GatedResourceLayout`
